src/test/regress/expected/limit_intermediate_size.out

```diff
@@ -16,7 +16,7 @@ SELECT cte.user_id, cte.value_2 FROM cte,cte2 ORDER BY 1,2 LIMIT 10;
 ERROR:  the intermediate result size exceeds citus.max_intermediate_result_size (currently 2 kB)
 DETAIL:  Citus restricts the size of intermediate results of complex subqueries and CTEs to avoid accidentally pulling large result sets into once place.
 HINT:  To run the current query, set citus.max_intermediate_result_size to a higher value or -1 to disable.
-SET citus.max_intermediate_result_size TO 9;
+SET citus.max_intermediate_result_size TO 17;
```

Comment:
Hmm, interesting. Is the intermediate result size on the coordinator > intermediate result size on the worker?

Reply:
I was not sure what the size should be. 
With two workers, I guess 4+4+1 is being used.
With coordinator, 4+4+4+1 = 13 did not work. So I put a bigger number for now :)